typing-practice-react/src/displayStats.js

```diff
@@ -0,0 +1,53 @@
+export function displayStats(userMistakesCount, userKeyTypeCount, seconds) {
+  if (userMistakesCount == 0 && userKeyTypeCount == 0) {
+    return 'CPM: 0 Wrong Chars: 0%';
+  }
+
+  let minutes = seconds / 60;
+  const charactersPerMinute = userKeyTypeCount / minutes;
+  
+  return `CPM: ${Math.round(charactersPerMinute)} Wrong Chars: ${Math.round((userMistakesCount / userKeyTypeCount) * 100)}%`;
+}
+
+export function displayTodayStats() {
+  return putTogetherStringsForStats('todayTotalSeconds', 'todayCharsTyped', 'todayMistypes', 'todayAmountOfSets')
+}
+
+export function displayTotalStats() {
+  return putTogetherStringsForStats('totalTotalSeconds', 'totalCharsTyped', 'totalMistypes', 'totalAmountOfSets');
+}
+
+export function toggleExtendedStats(extendedStatsContainer) {
+  if (extendedStatsContainer.style.display === 'none' || extendedStatsContainer.style.display === '') {
+    extendedStatsContainer.style.display = 'block';
+  }
+  else {
+    extendedStatsContainer.style.display = 'none';
+  }
+}
+
+function putTogetherStringsForStats(stringForSeconds, stringForCharsTyped, stringForMistypes, stringForAmountOfSets) {
+  let minutes = parseInt(localStorage.getItem(stringForSeconds)) / 60;
+  let cpmComputation = parseInt(localStorage.getItem(stringForCharsTyped)) / minutes;
+  let mistakeRatio = parseFloat(localStorage.getItem(stringForMistypes)) / parseFloat(localStorage.getItem(stringForCharsTyped));
+  let mistakeRatioWithThreeDecimalPlaces = mistakeRatio.toFixed(3);
```

Comment:
```suggestion
  let mistakeRationRounded = mistakeRatio.toFixed(3);
```

While others tend to use undescriptive variable names, you are too descriptive :D Keep them as long as necessary but as short as possible :)

Reply:
Okay 😁👍